README.md

```diff
@@ -92,7 +92,16 @@ resource in an EVRYTHNG account.
 
 In a nutshell, `evrythng.js` provides the following scopes. Once a scope is
 created it provides an appropriate API for the resources it can manage
-(see [API](#api) below):
+(see [API](#api) below).
+
+For `apiVersion:2` next scopes are avaliable:
+
+- `Operator`
+- `AccessToken`
```

Comment:
Add a description, something derived from the code comments on the class itself?

Reply:
fixed, thanks